### PR TITLE
Remove uneeded replacements from push-packages-to-obs 

### PR DESCRIPTION
**Pull request by juliogonzalez**
We now managed this at the prjconf

### CODE REVIEW

**Review by deneb-alpha — CHANGES_REQUESTED** [review]
can you please also add the changelog entry?
it could make sense to tag and release a new version

**Comment by juliogonzalez** [comment]
> can you please also add the changelog entry? it could make sense to tag and release a new version

Done, I didn't remember there was a spec and all the stuff.

File: uyuni-releng-tools.changes.juliogonzalez.50 [hunk 1]
```diff
@@ -0,0 +1 @@
+- Adapt to SUSE Manager 5.0
```
Comment (deneb-alpha) [inline uyuni-releng-tools.changes.juliogonzalez.50:1]:
```suggestion
- Adapt push-packages-to-obs to SUSE Manager 5.0
```

File: bin/push-packages-to-obs [hunk 2]
```diff
@@ -289,8 +289,8 @@ while read PKG_NAME; do
       if [ "${OSCAPI}" == "https://api.suse.de" ]; then
           # SUSE Manager settings
           VERSION=$(sed 's/^\([0-9]\+\.[0-9]\+\).*$/\1/' ${GIT_DIR}/rel-eng/packages/uyuni-base)
-          sed "s/^ARG INIT_IMAGE=.*$/ARG INIT_IMAGE=suse\/manager\/4.4\/init:latest/" -i $SRPM_PKG_DIR/Dockerfile
-          sed "s/^ARG INIT_BASE=.*$/ARG INIT_BASE=bci\/bci-init:15.4/" -i $SRPM_PKG_DIR/Dockerfile
+          sed "s/^ARG INIT_IMAGE=.*$/ARG INIT_IMAGE=suse\/manager\/5.0\/init:latest/" -i $SRPM_PKG_DIR/Dockerfile
+          sed "s/^ARG INIT_BASE=.*$/ARG INIT_BASE=bci\/bci-init:15.5/" -i $SRPM_PKG_DIR/Dockerfile
```
Comment (cbosdo) [inline bin/push-packages-to-obs:293]: We'll need the `bci-init:15.6` here

Comment (juliogonzalez) [reply bin/push-packages-to-obs:293]: Argh, true, even I myself mix this.

Comment (juliogonzalez) [reply bin/push-packages-to-obs]: Fixed

**Review by deneb-alpha — APPROVED** [review]
LGTM

File: uyuni-releng-tools.changes.juliogonzalez.50 [hunk 3]
```diff
@@ -0,0 +1,2 @@
+- Adapt push-packages-to-obs to SUSE Manager 5.0
```
Comment (cbosdo) [inline uyuni-releng-tools.changes.juliogonzalez.50:1]: May be this line can be dropped?

Comment (juliogonzalez) [reply uyuni-releng-tools.changes.juliogonzalez.50]: DOne

**Review by deneb-alpha — APPROVED** [review]
+1

**Review by cbosdo — APPROVED** [review]

**Pull request merged**